src/test/java/AgeVerificatorTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AgeVerificatorTest {
+
+    @Test
+    void isAgeLessThan18() {
```

Comment:
Nazwy testów powinny zaczynać się od should i na spokojnie można skrócić nazwę, by była podobna do nazwy metody, np. shouldBeAdult